### DISCUSSION TITLE
docstring for Python functions appearing in API documentation

### DISCUSSION THREAD

**Discussion by mitchbregs** [fastapi/fastapi#7577] [Questions] [answered]
### First check

* [x] I used the GitHub search to find a similar issue and didn't find it.
* [x] I searched the FastAPI documentation, with the integrated search.
* [x] I already searched in Google "How to X in FastAPI" and didn't find any information.

### Description

I am writing [`numpy` style docstrings](https://numpydoc.readthedocs.io/en/latest/format.html) and they are showing up in the `Parameter` section of `/docs` and `/redoc`. Is there a way to disable this functionality?  This is specific to my code, and not the API.

Thanks!

**Comment by retnikt** [comment, marked as answer]
You can put a form feed character escape sequence (`\f`) at the start of the docstring. See https://fastapi.tiangolo.com/advanced/path-operation-advanced-configuration/#advanced-description-from-docstring

**Comment by tiangolo** [comment]
Thanks for the help here @retnikt ! :clap: :bow: 

Thanks for reporting back and closing the issue @mitchbregs  :+1:
